src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/ParsingSimpleTypeModelBinder.cs

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Runtime.ExceptionServices;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding.Binders
+{
+    /// <summary>
+    /// An <see cref="IModelBinder"/> for decimal, double, float, and their <see cref="Nullable{T}"/> wrappers.
+    /// </summary>
+    public class ParsingSimpleTypeModelBinder : IModelBinder
```

Comment:
Hmmmmmmmmmmmmmmmmmmmmmmmmmmm

They say there are only two hard problems in computer science and we've already had lots of discussion about cache invalidation last week.

I'll throw out some ideas I had (all flawed)
- `DecimalTypeModelBinder`
- `NumericTypeModelBinder`
- `NumericModelBinder`